src/main/lrsql/system/webserver.clj

```diff
@@ -162,6 +163,8 @@
                           i/xapi-default-interceptors
                           http/create-server
                           http/start)]
+          (reset! openapi/oa-routes (::http/routes service))
```

Comment:
What's the rationale for using state like this vs making a function to build the interceptor?

Reply:
yeah, for simplicity's sake that makes sense, will change

Reply:
actually I take this back, it ends up being a circular dependency to pass it through `add-admin-routes`

Reply:
OK, was wondering about that. I'm gonna take a closer look at the code

Reply:
ended up adding another step to construction of `routes`